### PR TITLE
FIX: [Rest API Tool] Time delay when navigating to lengthy Header/Parameters

### PR DESCRIPTION
**Pull request by MiteshDarda**
## Description

Fixed the delay with lazy loading component, 
now the headers will only render when they are in view

## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [x] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [x] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build

## Have you tested locally?

- [ ] 👍 yes
- [ ] 🙅 no, because I am lazy

## Added tests?

- [ ] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md

### CODE REVIEW

**Comment by pratikshinde590** [comment]
Please attach the issue link

**Comment by pratikshinde590** [comment]
Please attach the issue link

**Comment by pratikshinde590** [comment]
Please attach the issue link

**Comment by pratikshinde590** [comment]
Please attach the issue link

**Comment by pratikshinde590** [comment]
Please attach the issue link

**Comment by pratikshinde590** [comment]
Please attach the issue link

**Comment by pratikshinde590** [comment]
Please attach the issue link

File: src/packages/@library/icons/Collections.svelte [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    fill={color}
+  />
```
Comment (itsmdasifraza) [inline src/packages/@library/icons/Collections.svelte:17]: icons not in use

**Comment by pratikshinde590** [comment]
Please attach the issue link

**Comment by pratikshinde590** [comment]
Please attach the issue link

**Comment by pratikshinde590** [comment]
Please attach the issue link

**Review by itsmdasifraza — APPROVED** [review]

**Review by Astitva877 — APPROVED** [review]

**Pull request merged**